### ISSUE TITLE
Add aliases

### ISSUE THREAD

**Issue by bacsog** [HadesArchitect/GrafanaCassandraDatasource#92]
When creating line-charts out of Cassandra queries, multiple lines can be put on the same panel. In this case both times the "ID value" field ends up being in the legend, even if I used to same ID value and wanted to query 2 different fields.

With Influx-based queries I could always rename these lines using aliases.

**Comment by HadesArchitect** [comment]
Sounds reasonable. Are you using query editor or configurator?

**Comment by bacsog** [comment]
Query editor for now. Now that you mention it I will log a ticket to make the configurator more user friendly. At the moment the query editor is friendlier. :)

**Comment by SteveHarveyUK** [comment]
Be great to get this feature.

**Comment by bacsog** [comment]
Until this gets developed, I came up with the following workaround:
The magic is in the first line of course.

```
select 		blobAsText(textAsBlob('ProviderA-ask'))
			,CAST(ask as double)
			,time
from                ProviderA.raw_prices
where 		symbol = '$instrument'
			AND time > $__timeFrom AND time < $__timeTo
ALLOW FILTERING;

```

**Comment by HadesArchitect** [comment]
@bacsog That's genius!

**Comment by HadesArchitect** [comment]
To be released as 2.0.0
